src/H5Dbtree.c

```diff
@@ -959,14 +1031,34 @@ H5D__btree_idx_get_addr(const H5D_chk_idx_info_t *idx_info, H5D_chunk_ud_t *udat
 } /* H5D__btree_idx_get_addr() */
 
 /*-------------------------------------------------------------------------
- * Function:	H5D__btree_idx_iterate_cb
+ * Function:    H5D__btree_idx_load_metadata
+ *
+ * Purpose:     Load additional chunk index metadata beyond the chunk index
+ *              itself. Currently a no-op.
+ *
+ * Return:      Non-negative on success/Negative on failure
+ *
+ *-------------------------------------------------------------------------
+ */
+static herr_t
+H5D__btree_idx_load_metadata(const H5D_chk_idx_info_t H5_ATTR_UNUSED *idx_info)
```

Comment:
Should this load the root node?

Reply:
So the root node is loaded collectively?

Reply:
Yes, at least for v1 btrees the metadata needed should already be loaded collectively after the "index open" call

Reply:
H5D__btree_idx_open() is a no-op. Does it happen in some other function?

Reply:
Ah yes sorry, on second look it doesn't occur as part of the index open call, it seems that the v1 B-tree root node exists directly at the file address for the chunk index and so no open really occurs. This may need to be revisited, but v1 btrees seem to be less straightforward than other index types

Reply:
I think you could  implement it the same way as the other indices with an address query of chunk 0 (though see my notes on the b-tree 2 implementation)